src/api/Controllers/BoreholeController.cs

```diff
@@ -273,6 +274,9 @@ await Context.Entry(hydrotest)
             boreholeGeometry.Id = 0;
         }
 
+        // remove navigation property
```

Comment:
PP: Kommentar braucht es nicht.

Reply:
mache ihn im nächsten PR weg, bin da auch noch in der Nähe unterwegs :)